framework/PageDashboard/PageDashboardChart.tsx

```diff
@@ -47,16 +91,38 @@ export function PageDashboardChart(props: {
             fixLabelOverlap
             // tickFormat={(date: string) => `${new Date(date).toLocaleDateString()}`}
             //  tickFormat={(n) => `${Math.round(n)}`}
+            label={settings && settings.xLabel}
+            style={{ axisLabel: { fontSize: 16 } }}
+          />
+          <ChartAxis
+            dependentAxis
+            showGrid
+            label={settings && settings.yLabel}
+            tickFormat={
+              settings && settings.onlyIntegerTicks
+                ? //eslint-disable-next-line @typescript-eslint/no-unsafe-return
+                  (t) => (Number.isInteger(t) ? t : '')
+                : undefined
+            }
+            style={{ axisLabel: { fontSize: 16 } }}
           />
-          <ChartAxis dependentAxis showGrid />
           <ChartStack>
-            {groups.map((group, index) => (
-              <ChartArea
-                key={index}
-                data={group.values.map((value) => ({ x: value.label, y: value.value }))}
-                interpolation="monotoneX"
-              />
-            ))}
+            {groups.map((group, index) =>
+              settings && settings.useLines ? (
+                <ChartLine
```

Comment:
Putting chart lines in a ChartStack makes the lines stack and not overlap.
The lines need to be outside of the ChartStack.
SEE:  https://github.com/ansible/ansible-ui/pull/781/files#diff-4b17c85c4a9d09cfb0e23bbcc5655cae1662a45515c215ca029119e5fd1d3d0cR87

Reply:
Fixed.